### PR TITLE
Update spacemandmm to 1.4

### PR DESCRIPTION
**Pull request by spookydonut**
## About The Pull Request
can't link tg pr rn

### CODE REVIEW

**Review by Ghommie — APPROVED** [review]
Hi big boss.

**Comment by silicons** [comment]
based

File: code/modules/vending/_vending.dm [hunk 1]
```diff
@@ -628,14 +628,14 @@ GLOBAL_LIST_EMPTY(vending_products)
...
+		if("takeoutcoin")
+			usr.put_in_hands(coin)
+			to_chat(usr, "<span class='notice'>You remove [coin] from [src].</span>")
+			coin = null
+			return
 			
-	else if("togglevoice" && panel_open)
-		shut_up = !shut_up
+		if("togglevoice" && panel_open)
+			shut_up = !shut_up
```
Comment (Ghommie) [inline code/modules/vending/_vending.dm]:
```suggestion
		if("togglevoice")
			if(panel_open)
				shut_up = !shut_up
```

**Review by Ghommie — CHANGES_REQUESTED** [review]
Hold on, turns out TravisCI is unable to fetch logs atm, but there was error in the compile step.

**Review by Ghommie — APPROVED** [review]

**Comment by Ghommie** [comment]
I consider this high priority as it patches some asinine issues too.

**Pull request merged**